puppet/zulip/manifests/base.pp

```diff
@@ -48,6 +48,7 @@
              'missedmessage_mobile_notifications',
              'outgoing_webhooks',
              'signups',
+             'slash_commands',
```

Comment:
I am actually not sure about this. It seems to be working without it, and I don't know why this is required. Can you please explain?